doc/API/centreon-api-v2.yaml

```diff
@@ -1265,50 +1389,38 @@ components:
              * `2` - CRITICAL
              * `3` - UNKNOWN
           example: 1
-    Acknowledgment.Service:
+    Acknowledgement.Service:
       allOf:
-        - $ref: '#/components/schemas/Acknowledgment.Host'
+        - $ref: '#/components/schemas/Acknowledgement.Host'
         - type: object
           properties:
             service_id:
               type: integer
               format: int64
               description: "ID of the service"
               example: 5
-    Acknowledgment.Host.Add:
+    Acknowledgement.Host.Add:
       type: object
       properties:
         comment:
           type: string
-          description: "Short description of the acknowledgment"
+          description: "Short description of the acknowledgement"
           example: "Acknowledged by admin"
-        host_id:
```

Comment:
shouldn't the hostId be specified ?

Reply:
it is specified in the URI now : /monitoring/hosts/{hostId}/acknowledgements